### ISSUE TITLE
CF7 fails to retrieve the list of apps in a space with a large number of apps

### ISSUE THREAD

**Issue by sebfoucault** [cloudfoundry/cli#2073]
**Please fill out the issue checklist below and provide ALL the requested information.**

- [X] I reviewed open and closed github issues that may be related to my problem.
- [X] I tried updating to the latest version of the CF CLI to see if it fixed my problem.
- [X] I attempted to run the command with `CF_TRACE=1` to help debug the issue.
- [X] I am reporting a bug that others will be able to reproduce.

**Describe the bug and the command you saw an issue with**
Trying to get the list of CF apps in a space with a large number of apps (600+), CF CLI v7.x fails while CF CLI v6.x succeeds.
Reported error refers to HTTP 414

**What happened**
CF CLI v7.x raises an error related to HTTP 414

```
CF> cf --version
cf version 7.1.0+4c3168f9a.2020-09-09
CF> cf apps
Getting apps in org myorg / space myspace as user@acme.com...

Error unmarshalling the following into a cloud controller error: <html>
<head><title>414 Request-URI Too Large</title></head>
<body>
<center><h1>414 Request-URI Too Large</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

**Expected behavior**
CF CLI v7.x should behave as 6.x does

```
CF> cf --version
cf version 6.53.0+8e2b70a4a.2020-10-01
CF> cf apps | tail -n +2 | wc -l
640
```

**Exact Steps To Reproduce**
1. Create hundreds of apps (exact number to be figured out)
1. Run `cf apps` in this space

**Provide more context**
- Windows, Windows WSL2 Ubuntu
- CF CLI version: `7.1.0+4c3168f9a.2020-09-09`
- CF API version: `3.88.0` 
- Problem seems related to the fact that `cf apps` passes all apps GUIDs in the query string when using `/v3/processes?app_guids=` (likely same cause as https://github.com/cloudfoundry/cli/issues/2062 for a different CF CLI command)

**Comment by reid47** [comment]
We recently released [v7.2.0](https://github.com/cloudfoundry/cli/releases/tag/v7.2.0), which should contain a fix for this issue. Thanks for your help!

**Comment by sebfoucault** [comment]
@reid47 - Thanks
